clearnode/rpc_router_private_test.go

```diff
@@ -699,6 +734,50 @@ func TestRPCRouterHandleCreateAppSession(t *testing.T) {
 
 		assertErrorResponse(t, ctx, "has challenged channels")
 	})
+	t.Run("ErrorNonZeroChannelAllocation", func(t *testing.T) {
```

Comment:
Why is this test duplicated here and in `app_session_service_test.go`?

Reply:
I created it for "just-in-case", but I agree, we can remove one of them

Reply:
Let's remove this one then.

Reply:
[3f5595b](https://github.com/erc7824/nitrolite/pull/414/commits/3f5595baa0b08bfcb751aec47ec3245d04930119)